rubicon-htb.js

```diff
@@ -254,6 +256,40 @@ function RubiconModule(configs) {
         return firstPartyData;
     }
 
+    function _getDigiTrustQueryParams() {
+        if (configs.digitrustId) {
+            _digiTrustId = configs.digitrustId;
+        }
+            function getDigiTrustId() {
```

Comment:
the spacing in this function is inconsistent.

Reply:
true I will edit it